test/cql/lwt_batch_test.cql

```diff
@@ -264,3 +269,7 @@ BEGIN BATCH
     UPDATE gh7113 SET int1 = 4, int2 = 8 WHERE part = 0 AND key = 9 IF lwt_trivial = null
     UPDATE gh7113 SET int1 = 0, int2 = 9 WHERE part = 0 AND key = 0 IF lwt_trivial = null
 APPLY BATCH;
+DROP TABLE gh7113;
+
+-- cleanup
+DROP KEYSPACE ks;
```

Comment:
This is not reliable, if any of the statements above fails, the test case will leave a keyspace behind, which IIUC will cause failure of subsequent tests that also use a keyspace with name `ks`.

Shouldn't the keyspace be provided by the framework, and use the appropriate partitioning mode (tablets vs vnodes) based on certain test configuration?

Reply:
https://github.com/scylladb/scylladb/pull/20858#issuecomment-2386294600